frontend/lib/parse-capi/sharing-urls.ts

```diff
@@ -0,0 +1,126 @@
+import { getNonEmptyString } from './';
```

Comment:
I'm temporarily importing this from `parse-capi/index.ts`, I think we should extract all these validation functions into their own module and will do this in a separate PR.`

Reply:
That's cool, there's a ticket to [sort all of this out](https://trello.com/c/AfMx5JSx) after AB Test launch. Be aware of the likely conflict with #181